config/a8c-for-agencies-development.json

```diff
@@ -19,7 +19,16 @@
 	"enable_all_sections": false,
 	"sections": {
 		"a8c-for-agencies": true,
-		"a8c-for-agencies-auth": true
+		"a8c-for-agencies-auth": true,
+		"a8c-for-agencies-marketplace": true,
+		"a8c-for-agencies-overview": true,
+		"a8c-for-agencies-plugins": true,
+		"a8c-for-agencies-purchases": true,
+		"a8c-for-agencies-referrals": true,
+		"a8c-for-agencies-resources": true,
+		"a8c-for-agencies-settings": true,
+		"a8c-for-agencies-sites": true,
+		"a8c-for-agencies-users": true
```

Comment:
I don't think we have to append every section with `a8c-for-agencies-`. We could probably do `navigation-item-` to be specific. What do you think?

Reply:
I believe it is necessary to append  `a8c-for-agencies-` here to uniquely identify sections related to the A4A portal.

Reply:
I mean everything here is related to the A4A portal, right? This is a nitpick; you can ignore it if you would like.